src/main/java/org/entur/lamassu/leader/entityupdater/VehicleFilter.java

```diff
@@ -30,57 +27,13 @@ class VehicleFilter implements Predicate<GBFSVehicle> {
 
   private final Logger logger = LoggerFactory.getLogger(this.getClass());
 
-  private final Map<String, PricingPlan> pricingPlans;
-  private final Map<String, VehicleType> vehicleTypes;
-
-  public VehicleFilter(
```

Comment:
Recreate this filter as a service, so we can reimplement the filters that were removed